### ISSUE TITLE
"Could not find peer" error when application starts

### ISSUE THREAD

**Issue by dmitryn** [status-im/status-react#3055]
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: 

When application starts it tries to connect to whisper node to get offline messages from. Sometimes that node isn't available so you end up seeing redbox error.

#### Expected behavior
[comment]: # (Describe what you expected to happen.)
No error is shown.

#### Actual behavior
Error is shown to the user

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

You don't get it every time obviously. Only when node is actually available (maybe down, maybe network issues, etc).

- Open Status
- Sign in

### Solution
[comment]: # (Please summarise the solution and provide a task list on what needs to be fixed.)
*Summary*: 

- [ ] Hide error from user as it's not informative to him
- [ ] Try re-connecting to peer/wnode. See @oskarth comments on proposed workflow below.
- [ ] Apply this solution to other parts dealing with wnode (basically to full initialize offline inboxing process)

### Additional Information

<img width="645" alt="screenshot 2018-01-16 09 03 19" src="https://user-images.githubusercontent.com/23836/34985616-b60b5f4e-fac5-11e7-8c25-58a6f9a1c604.png">

**Comment by oskarth** [comment]
Thanks for this @dmitryn, I believe we briefly talked about this in Slack a few weeks ago but I can't find it in history/notes. @adambabik do you recall? It was about using polling for AddPeer call or something like this, I believe. As a way to get around race issue (assuming that is the case, I guess it can fail for other reasons too).

As I recall it the idea would be something like: AddPeer -> poll to check if peer has been added (admin.peers or something like this) -> markTrustedPeer once this is done. markTrustedPeer should probably also handle this error more gracefully, at least client-side.

**Comment by adambabik** [comment]
@oskarth yeah, it's probably because we called `makeTrustedPeer` on an enode that was added via `addPeer` but not connected successfully yet.

Exactly, for now, polling between `AddPeer` and `markTrustedPeer` seems like the only option.

**Comment by dmitryn** [comment]
https://github.com/ethereum/go-ethereum/wiki/Connecting-to-the-network

More info on `admin.peers()`

**Comment by oskarth** [comment]
@dmitryn Do you want to write up a bounty for doing "Hide error from user as it's not informative to him"? And we can link here.

**Comment by dmitryn** [comment]
We now occasionally get  "Number of retries for fetching peers exceed..." error. Looks like 3 retries with 300ms interval is not always enough.

```
00:20 I/ReactNativeJS: INFO [status-im.protocol.handlers:285] - offline inbox: add-peer response enode://08d8eb6177b187049f6c97ed3f6c74fbbefb94c7ad10bafcaf4b65ce89c314dcfee0a8bc4e7a5b824dfa08b45b360cc78f34f0aff981f8386caa07652d2e601b@163.172.177.138:40404 true
00:20 E/BoostFramework: BoostFramework() : Exception_1 = java.lang.NoSuchMethodException: perfIOPrefetchStart [int, class java.lang.String]
00:20 I/chatty: uid=10172(im.status.ethereum) identical 4 lines
00:20 E/BoostFramework: BoostFramework() : Exception_1 = java.lang.NoSuchMethodException: perfIOPrefetchStart [int, class java.lang.String]
00:20 I/zygote: Deoptimizing void com.facebook.react.uimanager.NativeViewHierarchyManager.manageChildren(int, int[], com.facebook.react.uimanager.ViewAtIndex[], int[]) due to JIT inline cache
00:20 I/ReactNativeJS: INFO [status-im.protocol.handlers:294] - offline inbox: fetch-peers response [{:id "0f7c65277f916ff4379fe520b875082a56e587eb3ce1c1567d9ff94206bdb05ba167c52272f20f634cd1ebdec5d9dfeb393018bfde1595d8e64a717c8b46692f", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:47524", :remoteAddress "51.15.54.150:30303"}, :protocols {:shh "unknown"}} {:id "12d52c3796700fb5acff2c7d96df7bbb6d7109b67f3442ee3d99ac1c197016cddb4c3568bbeba05d39145c59c990cd64f76bc9b00d4b13f10095c49507dd4cf9", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:45268", :remoteAddress "51.15.63.110:30303"}, :protocols {:shh "unknown"}} {:id "482484b9198530ee2e00db89791823244ca41dcd372242e2e1297dd06f6d8dd357603960c5ad9cc8dc15fcdf0e4edd06b7ad7db590e67a0b54f798c26581ebd7", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:38631", :remoteAddress "51.15.75.138:30303"}, :protocols {:shh "unknown"}} {:id "7ab298cedc4185a894d21d8a4615262ec6bdce66c9b6783878258e0d5b31013d30c9038932432f70e5b2b6a5cd323bf820554fcb22fbc7b45367889522e9c449", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:48408", :remoteAddress "51.15.63.93:30303"}, :protocols {:shh "unknown"}} {:id "86ebc843aa51669e08e27400e435f957918e39dc540b021a2f3291ab776c88bbda3d97631639219b6e77e375ab7944222c47713bdeb3251b25779ce743a39d70", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:44003", :remoteAddress "212.47.254.155:30303"}, :protocols {:shh "unknown"}} {:id "88c2b24429a6f7683fbfd06874ae3f1e7c8b4a5ffb846e77c705ba02e2543789d66fc032b6606a8d8888eb6239a2abe5897ce83f78dcdcfcb027d6ea69aa6fe9", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:43139", :remoteAddress "163.172.157.61:30303"}, :protocols {:shh "unknown"}} {:id "a1ef9ba5550d5fac27f7cbd4e8d20a643ad75596f307c91cd6e7f85b548b8a6bf215cca436d6ee436d6135f9fe51398f8dd4c0bd6c6a0c332ccb41880f33ec12", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:41650", :remoteAddress "51.15.218.125:30303"}, :protocols {:shh "unknown"}} {:id "ce6854c2c77a8800fcc12600206c344b8053bb90ee3ba280e6c4f18f3141cdc5ee80bcc3bdb24cbc0e96dffd4b38d7b57546ed528c00af6cd604ab65c4d528f6", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:41011", :remoteAddress "163.172.153.124:30303"}, :protocols {:shh "unknown"}} {:id "d40871fc3e11b2649700978e06acd68a24af54e603d4333faecb70926ca7df93baa0b7bf4e927fcad9a7c1c07f9b325b22f6d1730e728314d0e4e6523e5cebc2", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:47185", :remoteAddress "51.15.132.235:30303"}, :protocols {:shh "unknown"}} {:id "e006f0b2dc98e757468b67173295519e9b6d5ff4842772acb18fd055c620727ab23766c95b8ee1008dea9e8ef61e83b1515ddb3fb56dbfb9dbf1f463552a7c9f", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:48342", :remoteAddress "212.47.237.127:30303"}, :protocols {:shh "unknown"}} {:id "e2a3587b7b41acfc49eddea9229281905d252efba0baf565cf6276df17faf04801b7879eead757da8b5be13b05f25e775ab6d857ff264bc53a89c027a657dd10", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:45018", :remoteAddress "51.15.45.114:30303"}, :protocols {:shh "unknown"}} {:id "ea37c9724762be7f668e15d3dc955562529ab4f01bd7951f0b3c1960b75ecba45e8c3bb3c8ebe6a7504d9a40dd99a562b13629cc8e5e12153451765f9a12a61d",
00:20 I/ReactNativeJS: INFO [status-im.protocol.handlers:300] - Peer enode://08d8eb6177b187049f6c97ed3f6c74fbbefb94c7ad10bafcaf4b65ce89c314dcfee0a8bc4e7a5b824dfa08b45b360cc78f34f0aff981f8386caa07652d2e601b@163.172.177.138:40404 is not registered. Retrying fetch peers.
00:21 D/StatusModule: callJail
00:21 D/StatusModule: ["functions","init"]
00:21 D/StatusModule: startCallJail
00:21 D/StatusModule: endCallJail
00:21 D/StatusModule: callJail
00:21 D/StatusModule: ["functions","init"]
00:21 D/StatusModule: startCallJail
00:21 D/StatusModule: endCallJail
00:21 D/StatusModule: callJail
00:21 D/StatusModule: ["commands",["phone",null],"short-preview"]
00:21 D/StatusModule: startCallJail
00:21 D/StatusModule: endCallJail
00:21 D/StatusModule: callJail
00:21 D/StatusModule: ["functions","init"]
00:21 D/StatusModule: startCallJail
00:21 D/StatusModule: endCallJail
00:21 I/ReactNativeJS: INFO [status-im.protocol.handlers:294] - offline inbox: fetch-peers response [{:id "0f7c65277f916ff4379fe520b875082a56e587eb3ce1c1567d9ff94206bdb05ba167c52272f20f634cd1ebdec5d9dfeb393018bfde1595d8e64a717c8b46692f", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:47524", :remoteAddress "51.15.54.150:30303"}, :protocols {:shh "unknown"}} {:id "12d52c3796700fb5acff2c7d96df7bbb6d7109b67f3442ee3d99ac1c197016cddb4c3568bbeba05d39145c59c990cd64f76bc9b00d4b13f10095c49507dd4cf9", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:45268", :remoteAddress "51.15.63.110:30303"}, :protocols {:shh "unknown"}} {:id "482484b9198530ee2e00db89791823244ca41dcd372242e2e1297dd06f6d8dd357603960c5ad9cc8dc15fcdf0e4edd06b7ad7db590e67a0b54f798c26581ebd7", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:38631", :remoteAddress "51.15.75.138:30303"}, :protocols {:shh "unknown"}} {:id "7ab298cedc4185a894d21d8a4615262ec6bdce66c9b6783878258e0d5b31013d30c9038932432f70e5b2b6a5cd323bf820554fcb22fbc7b45367889522e9c449", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:48408", :remoteAddress "51.15.63.93:30303"}, :protocols {:shh "unknown"}} {:id "86ebc843aa51669e08e27400e435f957918e39dc540b021a2f3291ab776c88bbda3d97631639219b6e77e375ab7944222c47713bdeb3251b25779ce743a39d70", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:44003", :remoteAddress "212.47.254.155:30303"}, :protocols {:shh "unknown"}} {:id "88c2b24429a6f7683fbfd06874ae3f1e7c8b4a5ffb846e77c705ba02e2543789d66fc032b6606a8d8888eb6239a2abe5897ce83f78dcdcfcb027d6ea69aa6fe9", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:43139", :remoteAddress "163.172.157.61:30303"}, :protocols {:shh "unknown"}} {:id "a1ef9ba5550d5fac27f7cbd4e8d20a643ad75596f307c91cd6e7f85b548b8a6bf215cca436d6ee436d6135f9fe51398f8dd4c0bd6c6a0c332ccb41880f33ec12", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:41650", :remoteAddress "51.15.218.125:30303"}, :protocols {:shh "unknown"}} {:id "ce6854c2c77a8800fcc12600206c344b8053bb90ee3ba280e6c4f18f3141cdc5ee80bcc3bdb24cbc0e96dffd4b38d7b57546ed528c00af6cd604ab65c4d528f6", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:41011", :remoteAddress "163.172.153.124:30303"}, :protocols {:shh "unknown"}} {:id "d40871fc3e11b2649700978e06acd68a24af54e603d4333faecb70926ca7df93baa0b7bf4e927fcad9a7c1c07f9b325b22f6d1730e728314d0e4e6523e5cebc2", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:47185", :remoteAddress "51.15.132.235:30303"}, :protocols {:shh "unknown"}} {:id "e006f0b2dc98e757468b67173295519e9b6d5ff4842772acb18fd055c620727ab23766c95b8ee1008dea9e8ef61e83b1515ddb3fb56dbfb9dbf1f463552a7c9f", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:48342", :remoteAddress "212.47.237.127:30303"}, :protocols {:shh "unknown"}} {:id "e2a3587b7b41acfc49eddea9229281905d252efba0baf565cf6276df17faf04801b7879eead757da8b5be13b05f25e775ab6d857ff264bc53a89c027a657dd10", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:45018", :remoteAddress "51.15.45.114:30303"}, :protocols {:shh "unknown"}} {:id "ea37c9724762be7f668e15d3dc955562529ab4f01bd7951f0b3c1960b75ecba45e8c3bb3c8ebe6a7504d9a40dd99a562b13629cc8e5e12153451765f9a12a61d",
00:21 I/ReactNativeJS: INFO [status-im.protocol.handlers:300] - Peer enode://08d8eb6177b187049f6c97ed3f6c74fbbefb94c7ad10bafcaf4b65ce89c314dcfee0a8bc4e7a5b824dfa08b45b360cc78f34f0aff981f8386caa07652d2e601b@163.172.177.138:40404 is not registered. Retrying fetch peers.
00:21 I/ReactNativeJS: INFO [status-im.protocol.handlers:294] - offline inbox: fetch-peers response [{:id "0f7c65277f916ff4379fe520b875082a56e587eb3ce1c1567d9ff94206bdb05ba167c52272f20f634cd1ebdec5d9dfeb393018bfde1595d8e64a717c8b46692f", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:47524", :remoteAddress "51.15.54.150:30303"}, :protocols {:shh "unknown"}} {:id "12d52c3796700fb5acff2c7d96df7bbb6d7109b67f3442ee3d99ac1c197016cddb4c3568bbeba05d39145c59c990cd64f76bc9b00d4b13f10095c49507dd4cf9", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:45268", :remoteAddress "51.15.63.110:30303"}, :protocols {:shh "unknown"}} {:id "482484b9198530ee2e00db89791823244ca41dcd372242e2e1297dd06f6d8dd357603960c5ad9cc8dc15fcdf0e4edd06b7ad7db590e67a0b54f798c26581ebd7", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:38631", :remoteAddress "51.15.75.138:30303"}, :protocols {:shh "unknown"}} {:id "7ab298cedc4185a894d21d8a4615262ec6bdce66c9b6783878258e0d5b31013d30c9038932432f70e5b2b6a5cd323bf820554fcb22fbc7b45367889522e9c449", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:48408", :remoteAddress "51.15.63.93:30303"}, :protocols {:shh "unknown"}} {:id "86ebc843aa51669e08e27400e435f957918e39dc540b021a2f3291ab776c88bbda3d97631639219b6e77e375ab7944222c47713bdeb3251b25779ce743a39d70", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:44003", :remoteAddress "212.47.254.155:30303"}, :protocols {:shh "unknown"}} {:id "88c2b24429a6f7683fbfd06874ae3f1e7c8b4a5ffb846e77c705ba02e2543789d66fc032b6606a8d8888eb6239a2abe5897ce83f78dcdcfcb027d6ea69aa6fe9", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:43139", :remoteAddress "163.172.157.61:30303"}, :protocols {:shh "unknown"}} {:id "a1ef9ba5550d5fac27f7cbd4e8d20a643ad75596f307c91cd6e7f85b548b8a6bf215cca436d6ee436d6135f9fe51398f8dd4c0bd6c6a0c332ccb41880f33ec12", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:41650", :remoteAddress "51.15.218.125:30303"}, :protocols {:shh "unknown"}} {:id "ce6854c2c77a8800fcc12600206c344b8053bb90ee3ba280e6c4f18f3141cdc5ee80bcc3bdb24cbc0e96dffd4b38d7b57546ed528c00af6cd604ab65c4d528f6", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:41011", :remoteAddress "163.172.153.124:30303"}, :protocols {:shh "unknown"}} {:id "d40871fc3e11b2649700978e06acd68a24af54e603d4333faecb70926ca7df93baa0b7bf4e927fcad9a7c1c07f9b325b22f6d1730e728314d0e4e6523e5cebc2", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:47185", :remoteAddress "51.15.132.235:30303"}, :protocols {:shh "unknown"}} {:id "e006f0b2dc98e757468b67173295519e9b6d5ff4842772acb18fd055c620727ab23766c95b8ee1008dea9e8ef61e83b1515ddb3fb56dbfb9dbf1f463552a7c9f", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:48342", :remoteAddress "212.47.237.127:30303"}, :protocols {:shh "unknown"}} {:id "e2a3587b7b41acfc49eddea9229281905d252efba0baf565cf6276df17faf04801b7879eead757da8b5be13b05f25e775ab6d857ff264bc53a89c027a657dd10", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:45018", :remoteAddress "51.15.45.114:30303"}, :protocols {:shh "unknown"}} {:id "ea37c9724762be7f668e15d3dc955562529ab4f01bd7951f0b3c1960b75ecba45e8c3bb3c8ebe6a7504d9a40dd99a562b13629cc8e5e12153451765f9a12a61d",
00:21 I/ReactNativeJS: INFO [status-im.protocol.handlers:300] - Peer enode://08d8eb6177b187049f6c97ed3f6c74fbbefb94c7ad10bafcaf4b65ce89c314dcfee0a8bc4e7a5b824dfa08b45b360cc78f34f0aff981f8386caa07652d2e601b@163.172.177.138:40404 is not registered. Retrying fetch peers.
00:21 I/ReactNativeJS: INFO [status-im.protocol.handlers:294] - offline inbox: fetch-peers response [{:id "0f7c65277f916ff4379fe520b875082a56e587eb3ce1c1567d9ff94206bdb05ba167c52272f20f634cd1ebdec5d9dfeb393018bfde1595d8e64a717c8b46692f", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:47524", :remoteAddress "51.15.54.150:30303"}, :protocols {:shh "unknown"}} {:id "12d52c3796700fb5acff2c7d96df7bbb6d7109b67f3442ee3d99ac1c197016cddb4c3568bbeba05d39145c59c990cd64f76bc9b00d4b13f10095c49507dd4cf9", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:45268", :remoteAddress "51.15.63.110:30303"}, :protocols {:shh "unknown"}} {:id "482484b9198530ee2e00db89791823244ca41dcd372242e2e1297dd06f6d8dd357603960c5ad9cc8dc15fcdf0e4edd06b7ad7db590e67a0b54f798c26581ebd7", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:38631", :remoteAddress "51.15.75.138:30303"}, :protocols {:shh "unknown"}} {:id "7ab298cedc4185a894d21d8a4615262ec6bdce66c9b6783878258e0d5b31013d30c9038932432f70e5b2b6a5cd323bf820554fcb22fbc7b45367889522e9c449", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:48408", :remoteAddress "51.15.63.93:30303"}, :protocols {:shh "unknown"}} {:id "86ebc843aa51669e08e27400e435f957918e39dc540b021a2f3291ab776c88bbda3d97631639219b6e77e375ab7944222c47713bdeb3251b25779ce743a39d70", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:44003", :remoteAddress "212.47.254.155:30303"}, :protocols {:shh "unknown"}} {:id "88c2b24429a6f7683fbfd06874ae3f1e7c8b4a5ffb846e77c705ba02e2543789d66fc032b6606a8d8888eb6239a2abe5897ce83f78dcdcfcb027d6ea69aa6fe9", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:43139", :remoteAddress "163.172.157.61:30303"}, :protocols {:shh "unknown"}} {:id "a1ef9ba5550d5fac27f7cbd4e8d20a643ad75596f307c91cd6e7f85b548b8a6bf215cca436d6ee436d6135f9fe51398f8dd4c0bd6c6a0c332ccb41880f33ec12", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:41650", :remoteAddress "51.15.218.125:30303"}, :protocols {:shh "unknown"}} {:id "ce6854c2c77a8800fcc12600206c344b8053bb90ee3ba280e6c4f18f3141cdc5ee80bcc3bdb24cbc0e96dffd4b38d7b57546ed528c00af6cd604ab65c4d528f6", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:41011", :remoteAddress "163.172.153.124:30303"}, :protocols {:shh "unknown"}} {:id "d40871fc3e11b2649700978e06acd68a24af54e603d4333faecb70926ca7df93baa0b7bf4e927fcad9a7c1c07f9b325b22f6d1730e728314d0e4e6523e5cebc2", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:47185", :remoteAddress "51.15.132.235:30303"}, :protocols {:shh "unknown"}} {:id "e006f0b2dc98e757468b67173295519e9b6d5ff4842772acb18fd055c620727ab23766c95b8ee1008dea9e8ef61e83b1515ddb3fb56dbfb9dbf1f463552a7c9f", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:48342", :remoteAddress "212.47.237.127:30303"}, :protocols {:shh "unknown"}} {:id "e2a3587b7b41acfc49eddea9229281905d252efba0baf565cf6276df17faf04801b7879eead757da8b5be13b05f25e775ab6d857ff264bc53a89c027a657dd10", :name "Geth/v1.7.3-stable/linux-amd64/go1.9.2", :caps ["eth/62" "eth/63" "les/1" "les/2" "shh/5"], :network {:localAddress "192.168.0.3:45018", :remoteAddress "51.15.45.114:30303"}, :protocols {:shh "unknown"}} {:id "ea37c9724762be7f668e15d3dc955562529ab4f01bd7951f0b3c1960b75ecba45e8c3bb3c8ebe6a7504d9a40dd99a562b13629cc8e5e12153451765f9a12a61d",
00:21 I/ReactNativeJS: INFO [status-im.protocol.handlers:300] - Peer enode://08d8eb6177b187049f6c97ed3f6c74fbbefb94c7ad10bafcaf4b65ce89c314dcfee0a8bc4e7a5b824dfa08b45b360cc78f34f0aff981f8386caa07652d2e601b@163.172.177.138:40404 is not registered. Retrying fetch peers.
00:21 E/ReactNativeJS: ERROR [status-im.protocol.handlers:232] - Number of retries for fetching peers exceed enode://08d8eb6177b187049f6c97ed3f6c74fbbefb94c7ad10bafcaf4b65ce89c314dcfee0a8bc4e7a5b824dfa08b45b360cc78f34f0aff981f8386caa07652d2e601b@163.172.177.138:40404
00:21 E/unknown:ReactNative: console.error: "ERROR [status-im.protocol.handlers:232] - Number of retries for fetching peers exceed enode://08d8eb6177b187049f6c97ed3f6c74fbbefb94c7ad10bafcaf4b65ce89c314dcfee0a8bc4e7a5b824dfa08b45b360cc78f34f0aff981f8386caa07652d2e601b@163.172.177.138:40404", stack:
00:21 E/unknown:ReactNative: cljs$core$IFn$_invoke$arity$3@12:349450
00:21 E/unknown:ReactNative: cljs$core$IFn$_invoke$arity$2@12:349247
00:21 E/unknown:ReactNative: cljs$core$IFn$_invoke$arity$2@12:354519
00:21 E/unknown:ReactNative: c@12:5614207
00:21 E/unknown:ReactNative: r@12:5614389
00:21 E/unknown:ReactNative: <unknown>@12:3970518
00:21 E/unknown:ReactNative: <unknown>@12:3989853
00:21 E/unknown:ReactNative: cljs$core$IKVReduce$_kv_reduce$arity$3@12:547475
00:21 E/unknown:ReactNative: _kv_reduce@12:159714
00:21 E/unknown:ReactNative: reduce_kv@12:269726
00:21 E/unknown:ReactNative: cljs$core$IFn$_invoke$arity$10@12:3987159
00:21 E/unknown:ReactNative: G__45690_45699@12:6844706
00:21 E/unknown:ReactNative: <unknown>@12:4674928
00:21 E/unknown:ReactNative: invoke_interceptor_fn@12:4630927
00:21 E/unknown:ReactNative: invoke_interceptors@12:4631228
00:21 E/unknown:ReactNative: execute@12:4632846
00:21 E/unknown:ReactNative: handle@12:4643355
00:21 E/unknown:ReactNative: re_frame$router$IEventQueue$_process_1st_event_in_queue$arity$1@12:4670624
00:21 E/unknown:ReactNative: re_frame$router$IEventQueue$_run_queue$arity$1@12:4654006
00:21 E/unknown:ReactNative: <unknown>@12:4657186
00:21 E/unknown:ReactNative: re_frame$router$IEventQueue$_fsm_trigger$arity$3@12:4661397
00:21 E/unknown:ReactNative: <unknown>@12:4655308
00:21 E/unknown:ReactNative: <unknown>@32:1772
00:21 E/unknown:ReactNative: b@32:628
00:21 E/unknown:ReactNative: w@32:992
00:21 E/unknown:ReactNative: callImmediates@32:3173
00:21 E/unknown:ReactNative: value@27:2778
00:21 E/unknown:ReactNative: <unknown>@27:1360
00:21 E/unknown:ReactNative: value@27:2606
00:21 E/unknown:ReactNative: value@27:1330
00:21 E/unknown:ReactNative: value@27:1265
```

**Comment by oskarth** [comment]
Should we reopen this @dmitryn and @yenda? Doesn't seem to be resolved based on https://github.com/status-im/status-react/issues/3333 if I understand the problem correctly

**Comment by dmitryn** [comment]
>Should we reopen this @dmitryn and @yenda? Doesn't seem to be resolved based on #3333 if I understand the problem correctly

@oskarth Yes, seems like "many retries fix" didn't help as mailserver node might be down  or maybe we have another edge case here.

**Comment by dmitryn** [comment]
Should be fixed by https://github.com/status-im/status-react/pull/3384
